lib/PuppeteerSharp/Page.cs

```diff
@@ -456,6 +457,18 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
             }
         }
 
+        public Task<dynamic> EvaluateExpressionAsync(string script) =>
```

Comment:
move the arrow to next line - https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/UrlHelperExtensions.cs#L366

Reply:
Cool, I was looking for examples :+1: